.gitlab-ci.yml

```diff
@@ -98,6 +98,7 @@ test:smoketests:linux:
     - apt-get update && apt-get install -q -y make liblzma-dev libssl-dev
   script:
```

Comment:
The dependencies should be installed in `before_script`.

Rationale: if I am running this test locally, I most likely want to install the dependencies once (iow, call `debian_setup`) but run the test many times (iow `test_main`).

Reply:
sure. modified.